examples/step-6/doc/results.dox

```diff
@@ -484,7 +484,43 @@ step-49, step-53, and step-54, among other tutorial programs that cover
 the issue. Information on curved domains can also be found in the
 documentation module on @ref manifold "Manifold descriptions".
 
-
+Why does it make sense to choose a mesh that tracks the internal
+interface? There are a number of reasons, but the most essential one
+comes down to what we actually integrate in our bilinear
+form. Conceptually, we want to integrate the term $A_{ij}^K=\int_K
+a(\mathbf x) \varphi_i(\mathbf x) \varphi_j(\mathbf x) ; dx$ as the
```

Comment:
actually, we integrate the *gradients* of the shape functions here in this program